ui/src/pages/Home.tsx

```diff
@@ -1,26 +1,21 @@
 import * as React from 'react';
 import styled from "styled-components";
-
-const Title = styled.h1`
-  font-size: 3em;
-  text-align: center;
-  color: palevioletred;
-`;
+import {colour, typography} from "../styles";
 
 const Button = styled.button`
   display: flex;    
-  color: palevioletred;
+  color: ${colour.accentColour};
```

Comment:
color vs colour lol

I'm of the mind to 🇺🇸  everything, but I'm fine with this way too.

Reply:
loool I thought long and hard about this and went with 🇨🇦 but i'm fine w/ either

Reply:
changed my mind 😅